app/views/users/index.html.erb

```diff
@@ -12,7 +13,7 @@
       <div class='card-body'>
         <h1 class="card-title text-decoration-none mt-4"><%= user.name %></h1>  
         <h6 class="card-text ,t-2">Number of posts: <span><%= user.posts_counter %></span></h6>
-        <%= link_to "See details", user_path(user), class: "btn btn-warning mt-10" %>
+        <%= link_to "See details", user_path(user), class: "btn-details" %>
        </div>
     </div>
   </div>
```

Comment:
- [ ] Please design the `Users#Index` page to look identical to the wireframe provided. In the wireframe, each user is rendered one per row while yours rendered two users per row. Kindly change this.

### Your Design

![image](https://github.com/ridabensalem/blog-app/assets/77106962/b0618b29-64ad-42b9-826d-7b5bf2be6dc0)

### Wireframe Template

![image](https://github.com/ridabensalem/blog-app/assets/77106962/195a33ca-2e6c-4dab-957d-d27f89ab42f3)

#### Hint: Kindly use the flex-direction: column rule of the tailwind CSS on the main container. 💻 🖱️